Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml

```diff
@@ -550,6 +550,39 @@
 
 # Cooked
 
+- type: entity
+  parent: BaseItem
+  id: MaterialSmileExtract
+  name: extract smile
+  description: It's a real panacea. But at what cost?
+  components:
+  - type: Food
+  - type: Extractable
+    grindableSolutionName: food
+  - type: FlavorProfile
+    flavors:
+      - sweet
+  - type: Sprite
+    netsync: false
+    sprite: Mobs/Aliens/slimes.rsi
+    layers:
+    - map: [ "enum.DamageStateVisualLayers.Base" ]
+      state: rainbow_slime_extract
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Omnizine
+          Quantity: 30
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Iron
+          Quantity: 10
+  - type: Item
+    size: 5
+  - type: StaticPrice
+    price: 1500 #It has so much Omnizin in it
```

Comment:
it should prob have the same value as the smile entity since it only drops 1

Reply:
done